lib/cacheman/backend/redis.ex

```diff
@@ -73,6 +81,6 @@ defmodule Cacheman.Backend.Redis do
     end)
   end
 
-  def ttl_command(ttl: :infinity), do: []
-  def ttl_command(ttl: ttl), do: ["PX", "#{ttl}"]
+  def ttl_command(:infinity), do: []
```

Comment:
Please correct me if I'm wrong(maybe we're not using this fn in apps), but this will break the existing API(passing keyword vs passing the ttl value).

Reply:
Although this function isn't explicitly defined as private, it is used only as a helper function within this module to transform one of the options into the redis cli command. Before, we had only one option, `ttl`, so passing it as a tuple worked fine. Now we have another option, `timeout`, so they aren't passed as tuples but as keyword list instead.

It won't break the API, as I changed the way this function is used internally [here](https://github.com/renderedtext/ex-cacheman/blob/vm/add_redix_timeout/lib/cacheman/backend/redis.ex#L60) and [here](https://github.com/renderedtext/ex-cacheman/blob/vm/add_redix_timeout/lib/cacheman/backend/redis.ex#L48).